Rdkafka.xs

```diff
@@ -1,11 +1,7 @@
 /* vim: set expandtab sts=4: */
-#define PERL_NO_GET_CONTEXT
```

Comment:
If we can keep `PERL_NO_GET_CONTEXT` that would be good for threaded perls
http://blogs.perl.org/users/nick_wellnhofer/2015/03/writing-xs-like-a-pro---perl-no-get-context-and-static-functions.html

Reply:
If I remember correctly, there was a problem with the logging callback function being called from a different thread in librdkafka, than the one connected to the perl interpreter context.